images/kube-proxy/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM registry.svc.ci.openshift.org/ocp/4.2:hyperkube AS builder
+RUN mkdir -p /tmp/build; \
+    cp /usr/bin/hyperkube /tmp/build/hyperkube
+
+FROM centos:7
+RUN INSTALL_PKGS="conntrack-tools" && \
+    yum install -y --setopt=tsflags=nodocs $INSTALL_PKGS && \
+    yum clean all && rm -rf /var/cache/*
+
+COPY --from=builder /tmp/build/hyperkube /usr/bin/
+COPY ./images/kube-proxy/scripts/iptables /usr/sbin/
+COPY ./images/kube-proxy/scripts/iptables-save /usr/sbin/
+COPY ./images/kube-proxy/scripts/iptables-restore /usr/sbin/
+COPY ./images/kube-proxy/scripts/ip6tables /usr/sbin/
+COPY ./images/kube-proxy/scripts/ip6tables-save /usr/sbin/
+COPY ./images/kube-proxy/scripts/ip6tables-restore /usr/sbin/
+COPY ./images/kube-proxy/scripts/iptables /usr/sbin/
+
+LABEL io.k8s.display-name="Kubernetes kube-proxy" \
+      io.k8s.description="Provides kube-proxy for external CNI plugins" \
+      io.openshift.tags="openshift,kube-proxy" \
+      io.openshift.build.versions="kubernetes=1.14.0"
```

Comment:
Hm... so we just have to remember to update that version number when we rebase? There's no way to automate that?

Reply:
hm, i'll check on that, i kept what was in origin/images/kube-proxy for now.

Reply:
I've removed that label, @deads2k is that build.versions label required?